contract_api/application/handler/service_handler.py

```diff
@@ -21,6 +21,7 @@ def save_offchain_attribute(event, context):
     org_id = event["pathParameters"]["orgId"]
     service_id = event["pathParameters"]["serviceId"]
     payload = json.loads(event["body"])
-    response = RegistryService(org_id=org_id, service_id=service_id).save_offchain_service_attribute(attributes=payload)
+    response = RegistryService(org_id=org_id, service_id=service_id).save_offchain_service_attribute(
+        new_offchain_attributes=payload)
```

Comment:
To make it more readable, let's add one more line
attributes = payload["attributes"]
response = RegistryService(org_id=org_id, service_id=service_id).save_offchain_service_attribute(
        new_offchain_attributes=attributes)